trace/google/cloud/gapic/trace/v1/trace_service_client.py

```diff
@@ -0,0 +1,310 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/google/googleapis/blob/master/google/devtools/cloudtrace/v1/trace.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google engineers.
+#
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+"""Accesses the google.devtools.cloudtrace.v1 TraceService API."""
+
+import collections
+import json
+import os
+import pkg_resources
+import platform
+
+from google.gax import api_callable
+from google.gax import config
+from google.gax import path_template
+import google.gax
+
+from google.cloud.gapic.trace.v1 import enums
+from google.cloud.proto.devtools.cloudtrace.v1 import trace_pb2
+from google.protobuf import timestamp_pb2
+
+_PageDesc = google.gax.PageDescriptor
+
+
+class TraceServiceClient(object):
+    """
+    This file describes an API for collecting and viewing traces and spans
+    within a trace.  A Trace is a collection of spans corresponding to a single
+    operation or set of operations for an application. A span is an individual
+    timed event which forms a node of the trace tree. Spans for a single trace
+    may span multiple services.
+    """
+
+    SERVICE_ADDRESS = 'cloudtrace.googleapis.com'
+    """The default address of the service."""
+
+    DEFAULT_SERVICE_PORT = 443
+    """The default port of the service."""
+
+    _PAGE_DESCRIPTORS = {
+        'list_traces': _PageDesc('page_token', 'next_page_token', 'traces')
+    }
+
+    # The scopes needed to make gRPC calls to all of the methods defined in
+    # this service
+    _ALL_SCOPES = ('https://www.googleapis.com/auth/cloud-platform',
+                   'https://www.googleapis.com/auth/trace.append',
+                   'https://www.googleapis.com/auth/trace.readonly', )
+
+    def __init__(self,
+                 service_path=SERVICE_ADDRESS,
+                 port=DEFAULT_SERVICE_PORT,
+                 channel=None,
+                 credentials=None,
+                 ssl_credentials=None,
+                 scopes=None,
+                 client_config=None,
+                 app_name=None,
+                 app_version='',
+                 lib_name=None,
+                 lib_version='',
+                 metrics_headers=()):
+        """Constructor.
+
+        Args:
```

Comment:
Is this the right documentation style for this repository?  This file doesn't quite match the style in enums.py